sci-mathematics/gimps/gimps-30.3.6.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd readme.gentoo-r1
```

Comment:
Also please sort this while at it, you need to inherit *tmpfiles.eclass* and use its function here.

Reply:
Done.